bin/schema_validator

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+# coding=utf-8
+
+import argparse
+import json
+
+from jsonschema import validate, Draft4Validator
+
+
+def parse_args():
+    desc = 'Validates a json file against a given json schema.'
+    parser = argparse.ArgumentParser(description=desc)
+    parser.add_argument('json_schema', type=str,
+                        help='Path to the json schema file')
+    parser.add_argument('json_file', type=str,
+                        help='Path to the json file')
+    return parser.parse_args()
+
+
+def read_json(json_file):
```

Comment:
Let's use the [existing generators to load JSON data](https://github.com/scrapinghub/skinfer/blob/master/skinfer/schema_inferer.py#L16), it's better than loading everything to memory.